ksqldb-common/src/main/java/io/confluent/ksql/properties/with/InsertIntoConfigs.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.properties.with;
+
+import org.apache.kafka.common.config.ConfigDef;
+
+/**
+ * 'With Clause' properties for 'INSERT INTO' statements.
+ */
+public final class InsertIntoConfigs {
+  public static final String QUERY_ID_PROPERTY = "ID";
```

Comment:
I chose `ID` as the best name for this property. However, if it is confusing with the `ID` keyword in the schema (i.e. `ID id int`), then we can consider these options.

```
with (id = 'my_id')
with (query_id = 'my_id')
with (name = 'my_id')
with (alias = 'my_id')
```

Reply:
I think originally I liked `id`, but now my preference is `query_id` 😂 it gives a little bit more clarity into what exactly it is. cc @colinhicks